### PR TITLE
LL-5333 Market

### PR DESCRIPTION
**Pull request by LFBarreto**
## 🦒 Context (issues, jira)
LL-MKT (MKT): init v3 provider

[LL-5333]

## 🖤  Expectations to reach

PR must pass CI, merge develop if conflicts, do not force push. Thanks!

- **on QA**: at least one of these two checkboxes must be checked:
  - [ ] a specific test planned is defined on Jira
  - [ ] this PR is covered by automatic UI test
- **on delivery**: at least one of these two checkboxes must be checked: 
  - [ ] Option 1: **no impact**: The changes of this PR have ZERO impact on the userland (invisible for users)
  - [ ] Option 2: **atomic delivery**: the changes is atomic and complete (no partial delivery)

PR must pass CI, merge develop if conflicts, do not force push. Thanks!

[LL-5333]: https://ledgerhq.atlassian.net/browse/LL-5333?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by valpinkman** [comment]
Reviewed the code and tested locally. The feature seems to be ready and pretty bug free (I have testes the happy paths... I'm  sure we can stress test it a bit more)

Code wise, it's kind of a pain to test because the builds hide the marketpage in sidebar (I cannot wait to have feature flags)

Other than that, looks good to me

https://user-images.githubusercontent.com/671786/148207987-884cf452-871f-402f-9e03-03cea6f6b0f7.mp4

File: src/renderer/screens/platform/Catalog.js [hunk 1]
```diff
@@ -74,6 +74,7 @@ const PlatformCatalog = () => {
   const history = useHistory();
 
   const { manifests } = usePlatformApp();
+  console.log(manifests);
```
Comment (valpinkman) [inline src/renderer/screens/platform/Catalog.js:77]: remove console.log

**Review by valpinkman — DISMISSED** [review]
Removing console.log and it will be ready for me

**Review by ghost — DISMISSED** [review]

**Review by valpinkman — DISMISSED** [review]

**Review by OlivierFreyssinet — APPROVED** [review]

**Pull request merged**